src/main/java/com/dev/gbk/controller/AuthController.java

```diff
@@ -51,16 +54,31 @@ public class AuthController {
 	@Autowired
     JwtTokenProvider tokenProvider;
 
+	@Autowired
+	GbkFeignClient gbkFeignClient;
+
+	@Autowired
+	Environment env;
+
 	@PostMapping("/signin")
 	public ResponseEntity<?> authenticateUser(@Valid @RequestBody LoginRequest loginRequest) {
-
 		Authentication authentication = authenticationManager.authenticate(
 				new UsernamePasswordAuthenticationToken(loginRequest.getUsernameOrEmail(), loginRequest.getPassword()));
 
+		ObjectMapper objectMapper = new ObjectMapper();
 		SecurityContextHolder.getContext().setAuthentication(authentication);
-
-		String jwt = tokenProvider.generateToken(authentication);
+		ReqGbkToken reqGbkToken = new ReqGbkToken(Integer.parseInt(env.getProperty("gbk.partner.id")), env.getProperty("gbk.partner.key"));
+		ResponseEntity<RespGbkToken> responseEntity = gbkFeignClient.getTokenGbk(reqGbkToken);
```

Comment:
Please make a service that implement the feign, so if there is timeout from external API we know about that